deepchem/models/callbacks.py

```diff
@@ -93,7 +95,24 @@ def __call__(self, model, step):
       if self._best_score is None or score < self._best_score:
         model.save_checkpoint(model_dir=self.save_dir)
         self._best_score = score
-    if model.wandb_logger is not None:
-      # Log data to Wandb
-      data = {'eval/' + k: v for k, v in scores.items()}
-      model.wandb_logger.log_data(data, step, dataset_id=id(self.dataset))
+        # Execute external logger checkpointing
+        for ext_logger in model.loggers:
+          if (self.name is None) or (not self.name):
+            # assign a default name to the callback based on dataset and metric
+            self.name = str(id(
+                self.dataset)) + "-" + self.metrics[self.save_metric].name
+          checkpoint_name = "callback-" + self.name + "-checkpoints"
+          # ensure score is positive if negated previously
+          checkpoint_score = score
+          if not self.save_on_minimum:
+            checkpoint_score = abs(checkpoint_score)
+          ext_logger.save_checkpoint(
+              self.save_dir,
+              model,
+              checkpoint_name,
+              self.metrics[self.save_metric].name,
+              checkpoint_score,
+              max_checkpoints_to_track=5,
+              checkpoint_on_min=self.save_on_minimum)
+    for ext_logger in model.loggers:
+      ext_logger.log_values(scores, step, location="eval/" + self.name)
```

Comment:
What about giving it a more informative name, for example `location=f'validation-{self.name}/'`?

Reply:
Sure, I've renamed it.